doc/pages/faces.asciidoc

```diff
@@ -2,183 +2,188 @@
 
 *StatusLine*::
-    face used for the status line
+    Face for the status line.
 
 *StatusLineMode*::
-    face used for the current mode except the normal mode
+    Face for the current mode, except normal mode.
 
 *StatusLineInfo*::
-    face used for special information
+    Face for special information.
 
 *StatusLineValue*::
-    face used for special values (numeric prefixes, registers, etc.)
+    Face for special values (numeric prefixes, registers, etc.).
 
 *StatusCursor*::
-    face used for the status line cursor
+    Face for the status line cursor.
 
 *Prompt*::
-    face used prompt displayed on the status line
+    Face for the prompt displayed on the status line.
 
 *BufferPadding*::
-    face applied on the `~` characters that follow the last line of a buffer
+    Face applied on the *~* characters that follow the last line of a buffer.
 
-=== Builtin highlighter faces
+=== Built-in highlighter faces
 
-The following faces are used by builtin highlighters if enabled.
+The following faces are used by built-in highlighters if enabled.
 (See <<highlighters#,`:doc highlighters`>>).
 
 *LineNumbers*::
-    face used by the `number-lines` highlighter
+    Face used by the *number-lines* highlighter.
 
 *LineNumberCursor*::
-    face used to highlight the line number of the main selection
+    Face used to highlight the line number of the main selection.
 
 *LineNumbersWrapped*::
-    face used to highlight the line number of wrapped lines
+    Face used to highlight the line number of wrapped lines.
 
 *MatchingChar*::
-    face used by the `show-matching` highlighter
+    Face used by the *show-matching* highlighter.
 
 *Whitespace*::
-    face used by the `show-whitespaces` highlighter
+    Face used by the *show-whitespaces* highlighter.
 
 *WrapMarker*::
-    face used by the `wrap -marker` highlighter
+    Face used by the *wrap-marker* highlighter.
 
 == Markup strings
 
-In certain contexts, Kakoune can take a markup string, which is a string
-containing formatting information. In these strings, the {facename}
-syntax will enable the face facename until another face gets activated,
-or the end of the string is reached.
+In certain contexts, Kakoune can understand markup strings, which are strings
+containing formatting information. In these strings, the {facename} syntax
+will enable the face facename until another face gets activated, or the end
+of the string is reached.
 
 For example, the following command displays the text "default" in the
-Default face, and "error" in the Error face:
+*Default* face, and "error" in the *Error* face:
 
 ----
 echo -markup 'default {Error}error{Default} default'
 ----
 
-Inside a markup string, a literal `{` character is written `\{`, and a
-literal backslash (`\`) that precedes a '{' character is escaped as well
+Inside a markup string, a literal `{` character is written `\{`, and a literal
+backslash `\` character that precedes a `{` character is escaped as well
```

Comment:
unrelated to your changes, so feel free to ignore: this documentation is misleading.
All backslashes in markup strings must be escaped.
For example,

	echo -markup \\

outputs only one backslash.

So I'd write something like

	Inside a markup string, a literal `{` character is written as `\{`, and a literal
	backslash (`\`) character is written as `\\`.

Reply:
I'll make your suggested change. But why the parentheses around the first backslash (`\`)? Aren't the backticks enough?

Reply:
> But why are there parentheses around the first backslash (\)? Aren't the backticks enough?

Backticks are not rendered by `:doc`, and on GitHub translate to grey background. The resulting "backslash \" is a pleonasm.

Reply:
Haha, now you're talking like me.

Note, pleonasms are about words, not symbols: "It _may_ be possible" instead of "It _is_ possible"; the **may** is already entailed by the **possible**, hence redundant.

Anyhow, shall I just use your suggestion for this passage?

> Inside a markup string, a literal `{` character is written as `\{`, and a literal
> backslash (`\`) character is written as `\\`.

Reply:
Yeah I think that's a good improvement

Reply:
Done